tests/NHSD.BuyingCatalogue.Solutions.Persistence.DatabaseTests/SolutionCapabilityRepositoryTests.cs

```diff
@@ -20,7 +21,7 @@ public class SolutionCapabilityRepositoryTests
 
         private const string SupplierId = "Sup 1";
 
-        private List<(Guid id, string name, string desc, string reference)> capDetails;
+        private static List<CapabilityDetails> _capDetails;
```

Comment:
The static surprised me here. As discussed this was put in due to an FXCop rule. This been the case I would like to suggest we change it to initialise the list here rather than in the setup.